data/campaigns/Under_the_Burning_Suns/scenarios/03_Stirring_in_the_Night.cfg

```diff
@@ -659,15 +659,11 @@
             speaker=Garak
             # po: Garak dies in this scenario, and the important part of this conversation is to tell the player that.
             #
-            # The English text is currently wrong, it's meant to say "dreams this night" not "dreams of this night", a
-            # typo that will be corrected in 1.18 but not 1.16. The next paragraph is talking about what the text should
-            # mean, not what it currently is.
-            #
             # This is said at LONGDARK4, with the moon on the schedule picture as if the next turn will be dawn. Garak
-            # has had the dreams tonight while they were camped here, he hasn't had premonitions beforehand; that's
+            # has had the dream tonight while they were camped here, he hasn't had premonitions beforehand; that's
             # probably just because of the nearby undead, not some prophetic vision. Neither Garak nor a first-time
             # player know that the scenario will extend the night indefinitely until the enemies are defeated.
-            message=_"I had dreams of this night Kaleh, full of gloom and darkness, my journey will end here one way or another."
+            message=_"I had a dream of this night Kaleh, full of gloom and darkness, my journey will end here one way or another."
```

Comment:
It's still got the "of" in. I thought we were going for exactly the wording in https://github.com/wesnoth/wesnoth/issues/7634#issuecomment-1553191149 .

Reply:
Sorry, thanks.

Reply:
Changing it to two sentences also seemed good. Sorry, I should have been clearer that it was more than a one word difference from Nemaara's version.